src/state/trajectory/selectors/index.ts

```diff
@@ -17,10 +14,7 @@ export const getIsNetworkedFile = createSelector(
         if (!simFile.name) {
             return false;
         }
-        return (
-            !!find(TRAJECTORIES, { id: simFile.name }) &&
-            isNetworkSimFileInterface(simFile)
-        );
+        return isNetworkSimFileInterface(simFile);
```

Comment:
a converted file is a networked trajectory we can no longer count on having all networked files in in our list of trajectories